src/category_theory/preadditive/biproducts.lean

```diff
@@ -5,10 +5,19 @@ Authors: Scott Morrison
 -/
 import tactic.abel
 import category_theory.limits.shapes.biproducts
+import category_theory.limits.preserves.shapes.biproducts
 import category_theory.preadditive
 
 /-!
-# Basic facts about morphisms between biproducts in preadditive categories.
+# Basic facts about biproducts in preadditive categories.
+
+In (or between) preadditive categories,
```

Comment:
Now that you have this, the "The remaining lemmas hold in any preadditive category." is probably redundant, I guess?